### PR TITLE
Rename Agent API parameters in some v3.3 methods

### PR DESCRIPTION
**Pull request by kogierman**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-6794-params-rename)
- [Jira](https://livechatinc.atlassian.net/browse/API-6794)

## 📓 Description

Rename parameter names in some v3.3 methods

### CODE REVIEW

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 1]
```diff
@@ -9,6 +9,21 @@ desc: "Changelog of the Agent Chat API"
...
+- `deactivate_chat` - `chat_id` => `id`
+- `follow_chat` - `chat_id` => `id`
+- `unfollow_chat` - `chat_id` => `id`
+- `transfer_chat` - `chat_id` => `id`
+- `update_chat_properties` - `chat_id` => `id`
+- `delete_chat_properties` - `chat_id` => `id`
+- `get_customer` - `customer_id` => `id`
+- `update_customer`  - `customer_id` => `id`
+- `ban_customer`  - `customer_id` => `id`
+
```
Comment (janowsiany) [inline content/messaging/agent-chat-api/changelog/index.mdx:26]: Can we keep changelog similar as in other PR: https://github.com/livechat/livechat-public-docs/pull/713

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/changelog/index.mdx:26]: @kogierman, could you adopt the new changelog style, please? See v3.2 changelog for reference (it's already on prod).
Here, I'd do the following entries:

> ### Chats
> In the **Deactivate Chat** ([Web] & [RTM]) method, the `chat_id` parameter was renamed to `id`.
> 
> ### Properties
> In the **Update Chat Proeprties** ([Web] & [RTM]) method, the `chat_id` parameter was renamed to `id`.
> 

...and so on.

If there's more than 1 change for a method, you can do a nested list, for example:

> ### Chats
> 
> The **Deactivate Chat** ([Web] & [RTM]) method:
>  - had the `chat_id` parameter renamed to `id`.

Comment (kogierman) [reply content/messaging/agent-chat-api/changelog/index.mdx:26]: sure, I'll align to new format

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 2]
```diff
@@ -9,6 +9,21 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3]
+
+### Chats
+- The **Deactivate Chat*** ([Web](/messaging/agent-chat-api/v3.3/#deactivate-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#deactivate-chat)) had the `chat_id` parameter renamed to `id`
+- The **Follow Chat*** ([Web](/messaging/agent-chat-api/v3.3/#follow-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#follow-chat)) had the `chat_id` parameter renamed to `id`
+- The **Unfollow Chat*** ([Web](/messaging/agent-chat-api/v3.3/#unfollow-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#unfollow-chat)) had the `chat_id` parameter renamed to `id`
+- The **Transfer Chat*** ([Web](/messaging/agent-chat-api/v3.3/#transfer-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#transfer-chat)) had the `chat_id` parameter renamed to `id`
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:18]: this one should be in ### Chat access

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 3]
```diff
@@ -9,6 +9,21 @@ desc: "Changelog of the Agent Chat API"
...
+- The **Update Chat Properties*** ([Web](/messaging/agent-chat-api/v3.3/#update-chat-properties) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#update-chat-properties)) had the `chat_id` parameter renamed to `id`
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:19]: These two should be in ### Properties

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 4]
```diff
@@ -9,6 +9,21 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3]
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:12]:
```suggestion
## [v3.3] - Developer preview
```

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 5]
```diff
@@ -9,6 +9,25 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3] - Developer preview
+
+### Chats
+- The **Deactivate Chat*** ([Web](/messaging/agent-chat-api/v3.3/#deactivate-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#deactivate-chat)) had the `chat_id` parameter renamed to `id`.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:15]: I've noticed in the UI that they have too many `***`

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 6]
```diff
@@ -9,6 +9,25 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3] - Developer preview
+
+### Chats
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:14]: we can add an empty line after each headline

**Pull request merged**